manifests/exec_restorecon.pp

```diff
@@ -0,0 +1,37 @@
+# selinux::exec_restorecon
+# 
+# A convenience wrapper around a restorecon exec
+#
+# Will execute after all other SELinux changes have been applied, but before
+# Anchor['selinux::end']
+# 
+# @param path The path to run restorecon on. Defaults to resource title.
+# @param recurse Whether restorecon should recurse. Defaults to true
+# @param refreshonly see the Exec resource
+# @param unless see the Exec resource
+# @param onlyif see the Exec resource
+#
+define selinux::exec_restorecon(
+  Stdlib::Absolutepath $path = $title,
+  Boolean $refreshonly = true,
+  Boolean $recurse = true,
+  Optional[String] $unless = undef,
+  Optional[String] $onlyif = undef,
+) {
+  include ::selinux
+  $command = $recurse ? {
+    true  => 'restorecon -R',
+    false => 'restorecon',
+  }
+
+  exec {"selinux::exec_restorecon ${path}":
+    path        => '/sbin:/usr/sbin',
```

Comment:
everytime i see path set on exec in forge modules i start thinking what is right: setting it here or leave it up to the user to set global defaults?

but this will work on all our supported platforms.